src/index.html

```diff
@@ -22,6 +22,349 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__link"
+      >
+        <img
+          src="images/logo.png"
+          alt="MOYO logo"
+          class="header__image"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#samsung"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#smartphones"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#laptops"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#gadgets"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#tablets"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#photo"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#video"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="product-image"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <div class="card__code">Product code: 195434</div>
```

Comment:
You need to use the semantic tag for text, fix it everywhere
```suggestion
        <p class="card__code">Product code: 195434</p>
```